src/context.rs

```diff
@@ -635,16 +635,16 @@ impl ConcreteBaseAudioContext {
         render_channel: Sender<ControlMessage>,
         offline: bool,
     ) -> Self {
-        // setup destination node with preferred number of channels
-        let destination_channel_count = Arc::new(AtomicUsize::new(number_of_channels as usize));
+        // placeholder for destination node channel count, will patch later
+        let tmp_dest_channel_count = Arc::default();
```

Comment:
I'm not too happy with the complexity of bootstrapping the audio context. I will rewrite this after the vq release